updated homepage for 2019

malawi-mike opened the pull request.

jdepree commented on index.php:
```diff
@@ -68,66 +68,72 @@
+			<img style="border:5px solid rgba(0, 0, 0, .85);" class="circle responsive-img" src="images/nyt_logo.jpg">
+				<h6 class="light center">industry praise</h6>
+					<br>
+			</div> -->
```
what is this?  Maybe remove this and the nyt_logo if we're not using it.

jdepree commented on index.php:
```diff
@@ -223,73 +231,22 @@ class="btn-large waves-effect waves-light light blue lighten-1" style="border-ra
-		      <div class="flow-text" style="padding:5% 5% 0 5%;font-size:20px;">*Based on 17 development metrics collected in each treatment and control village from 2014 (baseline) to 2017, with statistics calculated 
-				using a difference-in-differences model.  Not shown are small but statistically significant increases in motorcycles, TVs, and men and women in college.  Check out this <a href="impacts.php">page</a> for more info.  
-		      </div>
+		<!--  <div class="carousel-item white" href="#two!"> -->
```
Remove these if we're not using them anymore.

jdepree commented on index.php:
```diff
@@ -397,9 +352,9 @@ class="btn-large waves-effect waves-light light blue lighten-1" style="border-ra
+	<!--  <div class="icon-block" style="width:100%"><i class='material-icons left' style="font-size:20px;color:#03A9F4">timeline</i> = &nbsp;village data trends available
 	<br>
-	<i class='material-icons left' style="font-size:20px;color:#03A9F4">fiber_new</i> = &nbsp;data trends coming soon</div>
+	<i class='material-icons left' style="font-size:20px;color:#03A9F4">fiber_new</i> = &nbsp;data trends coming soon</div> -->
```
In general, version control works a lot better if you remove blocks instead of commenting them out.

jdepree commented:
I like the new header background, but it's a little too painful to read the text.  If we keep this background, we need to make it less of a strain to read.  Can you experiment with bordering/shadowing?

jdepree commented:
![Screen Shot 2019-03-15 at 8 40 09 AM](https://user-images.githubusercontent.com/2419351/54439206-f7d34880-46fd-11e9-9d80-d80f69dfc96f.png)

I think we need different styling on this - the contrast is a little weird.  Maybe darker text, text borders, darker background, different font -- it just looks rough the way it is.

jdepree commented:
I'm not sure we can entirely get away with the phrase "Live Track" -- I get what it's trying to convey, but it's not really English per se.  Maybe "Follow", though I get that that might lack the punch you're looking for.

malawi-mike commented:
cool. thanks for the feedback.  will push up some edits asap.

On Fri, Mar 15, 2019 at 10:45 AM Jeff DePree <notifications@github.com>
wrote:

> I'm not sure we can entirely get away with the phrase "Live Track" -- I
> get what it's trying to convey, but it's not really English per se. Maybe
> "Follow", though I get that that might lack the punch you're looking for.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Village-X-Org/Site-v2/pull/147#issuecomment-473313757>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AcyICjUoAQw1OwU76k6GPFvK6LnzYX5Vks5vW7H5gaJpZM4b2kfx>
> .
>


-- 
Mike Buckler, CEO & General Counsel
Village X Org <http://www.villagex.org>
mike@villagex.org
202-360-9931
<http://www.villagex.org>

malawi-mike commented:
made some more tweaks. check 'em out.

jdepree commented:
The contrast looks better on the top description, but I think that blue color is perhaps too strong for anything but very brief titles. Maybe we could just use dark-gray text and call it out with some sort of small flourish like these vertical bars?

![Screen Shot 2019-03-23 at 5 17 35 AM](https://user-images.githubusercontent.com/2419351/54861004-1dc6a180-4d2b-11e9-9d3d-5bb0e817e850.png)

jdepree commented:
I think the block after the asterisk at the bottom is a bit overwhelming - it feels like one of those drug commercials, but we're not legally obligated to say any of that stuff.  Maybe we can offload a big chunk of that onto the impacts page?  For now, I'll cut off "Not shown are small but statistically significant increases in motorcycles (+2 per village) and TVs (+2 per village). % change based on 2014 treatment village average.", but let me know if you'd rather go a different route.

malawi-mike commented:
just tweaked the home page a bit more.  liked most of your changes, but i would keep the "democracy disrupts extreme poverty" section as is, except for the blue text.  i would also use the full-sized banner image of the village building a nursery school.  looks like you reduced the image resolution, but, on my computer, the new resolution is blurry and less crisp than the other homepage images.

jdepree commented:
It looks like you changed it back to blue text -- did you mean to do that?

The picture is too large at present - 1.6mb for every page load.  We should at least cut it to fit, since it's currently square.  But it would be good to shrink the width as well -- currently, I think it's 2600 pixels, whereas the picture it replaced was 1280.  I dropped it to 1024 since that looked good on my computer, but maybe we could experiment with 1280?  Can you use Preview or another image editing app to cut it and see what works for you?  I also changed the opacity because it was looking a bit too dark on mine.

jdepree commented:
The previous banner photo height was 683

malawi-mike commented:
hmm.  didn't mean to change it back to blue text.  hope I did the commit
correctly.

will play with the banner pic resolution.  up early this morning, huh?

On Sat, Mar 23, 2019 at 9:10 PM Jeff DePree <notifications@github.com>
wrote:

> It looks like you changed it back to blue text -- did you mean to do that?
>
> The picture is too large at present - 1.6mb for every page load. We should
> at least cut it to fit, since it's currently square. But it would be good
> to shrink the width as well -- currently, I think it's 2600 pixels, whereas
> the picture it replaced was 1280. I dropped it to 1024 since that looked
> good on my computer, but maybe we could experiment with 1280? Can you use
> Preview or another image editing app to cut it and see what works for you?
> I also changed the opacity because it was looking a bit too dark on mine.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Village-X-Org/Site-v2/pull/147#issuecomment-475918702>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AcyICuPED3XL9U9C8pQWDXzk-7PDwx1sks5vZtCIgaJpZM4b2kfx>
> .
>


-- 
Mike Buckler, CEO & General Counsel
Village X Org <http://www.villagex.org>
mike@villagex.org
202-360-9931
<http://www.villagex.org>
